src/libmongoc/src/mongoc/mongoc-util.c

```diff
@@ -576,7 +576,7 @@ char *
 _mongoc_getenv (const char *name)
 {
 #ifdef _MSC_VER
-   char buf[1024];
+   char buf[2048];
```

Comment:
This needed to be bumped because the environment variable for the GCP private key data exceeds 1024 characters (`MONGOC_TEST_GCP_PRIVATEKEY`)